.env

```diff
@@ -4,3 +4,4 @@
 # For more info, see: https://nx.dev/concepts/inferred-tasks
 NX_ADD_PLUGINS=false
 PUBLIC_PACKAGE_VERSION=10.3.1-beta.20241023.1156
+ESLINT_USE_FLAT_CONFIG=false
```

Comment:
See https://eslint.org/docs/latest/use/migrate-to-9.0.0#-new-default-config-format-eslintconfigjs